ansible/inventory

```diff
@@ -190,6 +190,9 @@ staticiforme.torproject.org       # tor LDAP
 [no_nodeexp:children]
 no_passwd # no passwd => no nodeexp user ;-)
 
+[fastpath]
```

Comment:
Is there a reason to have it be part of the `fastpath` group?

Reply:
Should I use has_fastpath instead?